.drone.yml

```diff
@@ -45,7 +45,7 @@ pipeline:
       - /drone/.sbt
 
   build:
-    image: scalacenter/scala-docs:1.0
+    image: scalacenter/scala-docs:1.1
```

Comment:
I assume this new image has Nailgun or its deps inside? Python?

Reply:
It has only Python: See https://github.com/scalacenter/docker-scala-docs/pull/1